### PR TITLE
Skip specific bazel tests

### PR DESCRIPTION
**Pull request by vlad-nazarov**

### CODE REVIEW

**Comment by vlad-nazarov** [comment]
http://intel-ci.intel.com/eba6752a-5aa5-f111-acbd-8cdcd4b723a1

**Review by KulikovNikita — APPROVED** [review]

File: cpp/oneapi/dal/algo/svm/test/batch.cpp [hunk 1]
```diff
@@ -375,6 +375,7 @@ TEMPLATE_LIST_TEST_M(svm_batch_test,
                      "svm can classify linear separable surface with big margin",
                      "[svm][integration][batch][linear]",
                      svm_types) {
+    SKIP_IF(this->get_policy().is_cpu());
```
Comment (rlnx) [inline cpp/oneapi/dal/algo/svm/test/batch.cpp:378]: Why this test is skipped on CPU? If it's temporal, corresponding comment should be added, `// TODO: `

File: cpp/oneapi/dal/algo/svm/test/batch.cpp [hunk 2]
```diff
@@ -539,6 +540,7 @@ TEMPLATE_LIST_TEST_M(svm_batch_test,
                      "svm can classify any two labels",
                      "[svm][integration][batch][linear]",
                      svm_types) {
+    SKIP_IF(this->get_policy().is_cpu());
```
Comment (rlnx) [inline cpp/oneapi/dal/algo/svm/test/batch.cpp:543]: Why this test is skipped on CPU? If it's temporal, corresponding comment should be added, `// TODO:`

**Review by rlnx — APPROVED** [review]

**Pull request merged**